AlertaDengue/environment-dev.yml

```diff
@@ -0,0 +1,52 @@
+name: alertadengue
+channels:
+  - conda-forge
+  - defaults
```

Comment:
maybe you don't need `default` channel. but you can try it in a different PR.

Reply:
> maybe you don't need `default` channel. but you can try it in a different PR.

 Will be done!